modules/globebrowsing/src/ringscomponent.cpp

```diff
@@ -867,4 +876,50 @@ double RingsComponent::size() const {
     return _size;
 }
 
+ghoul::opengl::Texture* RingsComponent::ringTextureFwrd() const {
+    return _textureForwards.get();
+}
+
+ghoul::opengl::Texture* RingsComponent::ringTextureBckwrd() const {
+    return _textureBackwards.get();
+}
+
+ghoul::opengl::Texture* RingsComponent::ringTextureUnlit() const {
+    return _textureUnlit.get();
+}
+
+ghoul::opengl::Texture* RingsComponent::ringTextureColor() const {
+    return _textureColor.get();
+}
+
+ghoul::opengl::Texture* RingsComponent::ringTextureTransparency() const {
+    return _textureTransparency.get();
+}
```

Comment:
I'd probably say to rename these two:  `forwardTexture`, `backwardTexture`, `unlitTexture`, `colorTexture`, and `transparencyTexture`.  The "ring" is selfexplanatory from the call side, since it will be _ringComponent->forwardTexture()` and another "ring" would be double.

At the same time, we can also rename `_textureFwrdPath` to `_textureForwardPath` etc